app/src/main/java/me/ccrama/redditslide/SubmissionViews/PopulateSubmissionViewHolder.java

```diff
@@ -1723,11 +1710,11 @@ public void onPostExecute(Boolean b) {
             @Override
             protected Boolean doInBackground(Void... params) {
                 try {
-                    String toGet =
-                            new AccountManager(Authentication.reddit).reply(submission, reason);
+                    new AccountManager(Authentication.reddit).reply(submission, reason);
                     new ModerationManager(Authentication.reddit).remove(submission, false);
                     new ModerationManager(Authentication.reddit).setDistinguishedStatus(
-                            Authentication.reddit.get(toGet).get(0), DistinguishedStatus.MODERATOR);
```

Comment:
Since I run the formatter every time, so fixes get lost in the sea of changes, here is the ONLY functionality change that fixed this.

"toGet" is the id of the submission, but not the name. When Authentication.reddit.get(toGet), runs, it gets the value based on the id. Since this id was not correctly, an index out of bounds of thrown. The post WAS correctly removed prior but the app was crashed.

Passing in `submission.fullName()` resolved this as that is the correct id the method was looking for.